src/Components/TodoForm.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+function TodoForm({ addTodo }) {
+    const [value, setValue] = useState("");
+
+    const updateValue = e => {
+        setValue(e.currentTarget.value);
+    };
+
+    const handleSubmit = e => {
+        e.preventDefault();
+        let newTodo = {
+            id: new Date().getTime(),
+            task: value,
+            completed: false
+        };
+        addTodo(newTodo);
+        setValue("");
+    };
+
+    return (
+        <form onSubmit={handleSubmit}>
```

Comment:
Good use of form here! I didn't use this for ages and its stupid coz it still works fine if you don't use it haha, so its hard to know that its wrong not to use it! Form actually handles a lot of extra stuff like accessibility like using the keyboard etc